src/application-manager.coffee

```diff
@@ -17,7 +17,7 @@ Images = require './compose/images'
 Networks = require './compose/networks'
 Volumes = require './compose/volumes'
 
-Proxyvisor = require './proxyvisor'
+Proxyvisor = require './proxyvisor/proxyvisor'
```

Comment:
If you rename proxyvisor.coffee into index.coffee then here you can still `require './proxyvisor'`